url-updater/src/main/java/com/devonfw/tools/ide/url/updater/intellij/IntellijUrlUpdater.java

```diff
@@ -67,9 +124,27 @@ protected String getTool() {
   }
 
   @Override
-  protected String mapVersion(String version) {
+  protected String doGetVersionUrl() {
 
-    // TODO remove this hack, see #1132
-    return super.mapVersion(version.replace("-", "."));
+    return getVersionBaseUrl() + "/" + JSON_URL;
+  }
+
+  /**
+   *
+   * @return
```

Comment:
Indeed it is pointless to have empty JavaDoc. Either you override a method and use @Override. Then you will inherit the JavaDoc from the method declarion. Or you declare the method here and then it makes sense to document what it does (returns). If you do not want to do it, it is better to omitt JavaDoc entirely instead of adding 5 pointless JavaDoc lines as pure clutter with absolutely no additional information.